client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -224,11 +226,48 @@ function PurchaseMetaOwner( { owner } ) {
 
 function PurchaseMetaPrice( { purchase } ) {
 	const translate = useTranslate();
+	const overallState = useSelector( ( state ) => state );
```

Comment:
We might as well use `useSelector( state => getDIFMTieredPurchaseDetails( state, purchase.id )` here. Otherwise, this component will be re-rendered if any part of the global state changes.